crates/router/src/connector/aci/transformers.rs

```diff
@@ -203,6 +312,28 @@ pub struct ErrorParameters {
     pub(super) message: String,
 }
 
+fn method_data(redirection_data: AciRedirectionData) -> services::Method {
+    // Check if method exists in 3DS
+    if let Some(method_param) = redirection_data
+        .parameters
+        .iter()
+        .find(|param| param.name == *"method")
+    {
+        // Parse the parameter value as Method enum
+        if let Ok(method) = &method_param.value.parse::<services::Method>() {
+            return *method;
+        }
+    }
+
+    // Check if method exists in Redirects
+    if let Some(method) = redirection_data.method {
+        return method;
+    }
+
+    // Default to POST if no method is specified which is rare/edge case
+    services::Method::Post
```

Comment:
```suggestion
redirection_data.method.unwrap_or(services::Method::Get)
```

Default should be Get not Post